common/app/views/fragments/commercial/badge.scala.html

```diff
@@ -4,8 +4,10 @@
 @import views.support.Commercial.{isPaidContent, isOnDarkBackground}
 
 @for(commercial <- page.metadata.commercial; branding <- commercial.branding(Edition(request))) {
-    @pageLogo(branding,
-              content.tags.isInteractive,
-              isOnDarkBackground(content, isPaidContent(page))
-    )
+    @if(branding.sponsorName != "No Sponsor") {
+        @pageLogo(branding,
```

Comment:
it awaits PR in `commercial-shared` project: https://github.com/guardian/commercial-shared/pull/42

so we can use `isUnsponsored ` function instead of hardcoding "No Sponsor"